modules/apps/map/map-common/.npmbridgerc

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
cc @jbalsas @izaera this isn't needed anymore when we namespace the package, right? Or am I off in thinking this? 

I've read the bridge docs more times than I care to admit, and I still don't really ever feel confident around it 🙀

Reply:
We should remove bridges wherever we can and use proper imports instead. Bridges were intended to maintain compatibility with old code that didn't use version numbers, but we shouldn't introduce more code like that.

That said, I'm not sure why they were used here. Are we really sure we can get rid of them? I've seen some DDM code, for example, doing magic things when it comes to loading modules...

Reply:
I analyzed why `<aui:script require>` was not working without version numbers and it turned out that it was a feature we forced long ago :sweat_smile: (see https://github.com/liferay-frontend/liferay-portal/pull/1511#issue-745657846).

So we don't want it to work but instead, put JS code where it belongs (.js files) and use the magic `npmResolvedPackageName` to bootstrap JS code from JSPs.

I've seen some places in portal where people are leveraging bridges to do that bootstrap and avoid internalizing JS code in .js files. We should be aware of that and don't let it happen any more. I'll write about it in more detail in slack.